ledger-test-tool/tool/BUILD.bazel

```diff
@@ -0,0 +1,280 @@
+    "--pem $$(rlocation $$TEST_WORKSPACE/$(rootpath //test-common/test-certificates:client.pem))",
+    "--concurrent-test-runs=1",  # lowered from default #procs to reduce flakes - details in https://github.com/digital-asset/daml/issues/7316
+    "--timeout-scale-factor=2",  # increased to reduce flakes particularly wrt timeouts in TransactionService*IT tests
+]
+
+# conformance tests for default and latest
+#  those test the conformance test it-self for stable features
+conformance_test(
+    name = "conformance-test",
+    extra_data = conformance_test_extra_data_base + ["//canton:canton_deploy.jar"],
+    extra_runner_args = ["7000"],
+    lf_versions = [
+        "default",
+        "latest",
+    ],
+    ports = conformance_test_ports,
+    runner = "@//bazel_tools/client_server/runner_with_health_check",
+    server = ":canton-test-runner-with-dependencies",
+    server_args = conformance_server_args_base + [
+        "-C",
+        ",".join([
+            "dev-mode=no",
+            "_shared.participant_ledger-api_tls.minimum-server-protocol-version=TLSv1.3",
+        ]),
+    ],
+    test_tool_args = conformance_test_tool_args_base + [
+        "--exclude=" + ",".join([
+            "ClosedWorldIT",  # Canton currently fails this test with a different error (missing namespace in "unallocated" party id)
+            # dynamic config management not supported by Canton
+            "ConfigManagementServiceIT",
+            "LedgerConfigurationServiceIT",
+            "ParticipantPruningIT",  # pruning not supported in Canton Community
+            "CommandDeduplicationPeriodValidationIT:OffsetPruned",  # requires pruning not available in canton community
+            "ActiveContractsServiceIT:AcsBeforePruningOffsetIsDisallowed",  # requires pruning not available in canton community
+            "ActiveContractsServiceIT:AcsAtPruningOffsetIsAllowed",  # requires pruning not available in canton community
+        ]),
+        "--additional=TLSOnePointThreeIT",
+    ],
+) if not is_windows else None
+
+# this tests TLSAtLeastOnePointTwoIT
+conformance_test(
+    name = "conformance-test-tls1.2-or-newer",
+    extra_data = conformance_test_extra_data_base + ["//canton:canton_deploy.jar"],
+    extra_runner_args = ["7000"],
+    lf_versions = ["latest"],
+    ports = conformance_test_ports,
+    runner = "@//bazel_tools/client_server/runner_with_health_check",
+    server = ":canton-test-runner-with-dependencies",
+    server_args = conformance_server_args_base + [
+        "-C",
+        ",".join([
+            "dev-mode=no",
+        ]),
+    ],
+    test_tool_args = conformance_test_tool_args_base + ["--include=TLSAtLeastOnePointTwoIT"],
+) if not is_windows else None
+
+# conformance tests for preview and dev
+#  those test the conformance test it-self for preview/dev features and the patched canton jar
+conformance_test(
+    name = "conformance-test",
```

Comment:
same name used twice

Reply:
conformance_test suffix the test with the LF version, so this actually does not conflict. 

```
//ledger-test-tool/tool:conformance-test-1.15                            PASSED in 462.9s
//ledger-test-tool/tool:conformance-test-1.dev                           PASSED in 444.0s
//ledger-test-tool/tool:conformance-test-tls1.2-or-newer-1.15            PASSED in 72.8s
//ledger-test-tool/tool:test-extract                                     PASSED in 1.8s
```

Anyway, do you prefer I use something like `conformance-test-dev` ?